ARMeilleure/Translation/Translator.cs

```diff
@@ -406,7 +437,7 @@ internal static void EmitRejitCheck(ArmEmitterContext context, out Counter<uint>
 
             Operand lblEnd = Label();
 
-            Operand address = Const(ref counter.Value, Ptc.CountTableIndex);
+            Operand address = Const(ref counter.Value, Ptc.CountTableSymbol);
```

Comment:
```suggestion
            Operand address = !context.HasPtc ? Const(ref counter.Value) : Const(ref counter.Value, Ptc.CountTableSymbol);
```
Otherwise Operand will be Relocatable and we may lose optimisations.